style.css

```diff
@@ -0,0 +1,1498 @@
+}
+
+.smaller {
+  font-size: 13px;
+}
+
+.small {
+  font-size: 14px;
+}
+
+.gray {
+  background-color: gray;
+}
+
+.mr-30 {
+  margin-right: 30px;
+}
+
+.flex {
+  display: flex;
+}
+
+.flex-column {
+  flex-direction: column;
+}
+
+.justify-content-center {
+  justify-content: center;
+}
+
+.align-items-center {
+  align-items: center;
+}
+
+.align-items-baseline {
+  align-items: baseline;
+}
+
+.align-items-flex-end {
+  align-items: flex-end;
+}
+
+.space-between {
+  justify-content: space-between;
+}
+
+.space-around {
+  justify-content: space-around;
+}
+
+.flex-end {
+  justify-content: flex-end;
+}
+
+.grid {
+  display: grid;
+}
+
+.bigger {
+  font-size: 56px;
+}
+
+/* Nav Section */
+
+.hamburger-icon-mobile {
+  display: none;
+}
+
+.search-icon-mobile {
+  display: none;
+}
+
+.tablet-tnw-logo {
+  display: none;
+}
+
+.latest-funding a {
+  text-decoration: none;
+  color: #f42;
+}
+
+.footer-links-social a {
+  background-color: #333;
+  padding: 12px;
+  border-radius: 50%;
+  margin: 10px;
+  display: flex;
+}
+
+.footer-links-menu a {
+  color: #aaa;
+  margin: 20px 6px;
+  text-decoration: none;
+  font-size: 12px;
+  font-family: "Shentox", serif;
+}
+
+.latest-news h2 a {
+  text-decoration: none;
+  font-family: "Avalon", serif;
+  font-size: 28px;
+  color: #f42;
+}
+
+.card-1-mobile-text h3 a {
+  color: black;
+  text-decoration: none;
+}
+
+.latest-funding h2 a {
+  text-decoration: none;
+  font-size: 28px;
+  color: #f42;
+  font-family: "Avalon", sans-serif;
+}
+
+.card-3 li a {
+  color: #555;
+  font-size: 13px;
+}
+
+.latest-deals h2 a {
+  font-family: "Avalon", sans-serif;
+  font-size: 28px;
+  text-decoration: none;
+  color: #f42;
+}
+
+.footer-links-social li a {
+  border: transparent 1px solid;
+}
+
+.card-3 a:last-child {
+  text-decoration: none;
+  color: #aaa;
+}
+
+.footer-links-menu a:hover {
+  color: white;
+}
+
+.card-3 li a:hover {
+  color: black;
+}
+
+.latest-funding h2 a:hover {
+  color: black;
+}
+
+.latest-news h2 a:hover {
+  color: black;
+}
+
+.card-1-mobile-text h3 a:hover {
+  color: #f42;
+  text-decoration: underline;
+}
+
+.card-3 h2 a:first-child {
+  font-family: "Avalon", sans-serif;
+  font-size: 28px;
+  text-decoration: none;
+  color: #f42;
+  line-height: 30.8px;
+  letter-spacing: -0.42px;
+}
+
+.latest-deals h2 a:hover {
+  color: black;
+}
+
+.footer-links-social li a:hover {
+  background-color: #131313;
+  border: 1px solid #555;
+}
+
+.card-3 h2 a:first-child:hover {
+  font-family: "Avalon", sans-serif;
+  font-size: 28px;
+  text-decoration: none;
+  color: black;
+}
+
+#first-nav a {
+  color: white;
+  margin-left: 12px;
+}
+
+#first-nav ul li a {
+  font-family: "ARS Maquette", sans-serif;
+  color: #fff9;
+}
+
+#first-nav a:hover {
+  color: rgb(255, 255, 255);
+}
+
+#second-nav a:first-child {
+  margin-left: 28.5px;
+  color: #aaa;
+}
+
+#second-nav a:hover {
+  color: #131313;
+}
+
+#first-nav ul:first-child li:first-child a {
+  color: white;
```

Comment:
.card-3 li a {## Using Inefficient CSS Selectors[1]

When you are using selectors, the longer it is, the higher specificity it will comprise. This can break the CSS cascade and mess up the styles of your site.

This can be avoided by not using IDs within the selectors as they are directly specific.

 - [ ]  Please add a class to `<a>` element` and delete the left unnecessary rules

**Example:**
```CSS
/* before */
#first-nav ul:first-child li:first-child a {

/* after */
.anchortag-class {
```
<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_